.travis.yml

```diff
@@ -16,8 +16,9 @@ script:
 before_deploy:
   - npm run build
 deploy:
-  skip_cleanup: true
   provider: npm
+  edge: true
+  cleanup: true
   email: $NPM_EMAIL
   api_key: $NPM_KEY
   on:
```

Comment:
https://docs.travis-ci.com/user/deployment/releases/ official doc recommended using v2 
So I used https://docs.travis-ci.com/user/deployment-v2 for the GitHub tags release